packages/mc-scripts/src/utils/credentials-storage.js

```diff
@@ -0,0 +1,54 @@
+const { existsSync, writeFileSync, readFileSync, mkdirSync } = require('fs');
+const cookie = require('cookie');
+const homedir = require('os').homedir();
+
+const dirPath = `${homedir}/.mcscriptsrc`;
+const filename = 'credentials.json';
+const filePath = `${dirPath}/${filename}`;
+
+class CredentialsStorage {
+  constructor(cloudIdentifier) {
+    this.cloudIdentifier = cloudIdentifier;
+  }
+
+  _updateCredentialsFile(credentials) {
+    writeFileSync(filePath, JSON.stringify(credentials), { encoding: 'utf8' });
```

Comment:
How about stringifying it with formatting like `JSON.stringify(credentials, null, 2)` or `JSON.stringify(credentials, null, 4)`?

Reply:
Is there a reason you are suggesting adding extra white spaces?

Reply:
Just for readability. What do you reckon?

Reply:
No one is expected to read the file though. It's just for storing and retrieving the users session data.

Reply:
Agreed. It was just an idea and only meant for the rare cases when something requires debugging.
Every configuration json file saved in my `~` is formatted.